server/s3_put_multiobject_copy_action.cc

```diff
@@ -389,7 +423,7 @@ void S3PutMultipartCopyAction::copy_part_object() {
         std::bind(&S3PutMultipartCopyAction::copy_part_object_failed, this));
     f_success = true;
   }
-  catch (const std::exception &ex) {
+  catch (const std::exception& ex) {
```

Comment:
Look out for the clang errors fixed last week
https://github.com/Seagate/cortx-s3server/pull/1688/files# server/s3_put_multiobject_copy_action.cc line 392

Reply:
I have already formatted file with clang command.